lib/utils.js

```diff
@@ -2,7 +2,7 @@ const joi = require('@hapi/joi');
 const { ObjectId } = require('bson');
 const buildPropertySchema = (name, propertyDef) => { // eslint-disable-line complexity
   let schema = null;
-  switch (propertyDef.type.trim().toLowerCase()) {
+  switch ((propertyDef.type || 'any').trim().toLowerCase()) {
```

Comment:
this I changed cause I was getting a error like cannot trim undefined because of this property https://github.com/Losant/cosa/pull/43/files#diff-caf2204ca01a3a58fc231622387fd87057cbaa0e4828c48d5f1b463ab13f57d1R21